README.md

```diff
@@ -31,7 +31,7 @@ A nodeJS youtube jukebox application. Great for parties and stuff :-)
 
 #### OS X version 10.11 and up
 1. FFmpeg - Install using any of three options which can be found [here](https://superuser.com/a/624562). 
-1. Speaker - Install Node from [here](https://nodejs.org/en/download/)(or you can use Homebrew to install). Then run `sudo npm install speaker`.
+1. Speaker - Install Node from [here](https://nodejs.org/en/download/) (or you can use Homebrew to install). Then run `sudo npm install speaker`.

```

Comment:
This isn't really necessary. We can just put speaker in package.json nearer the time

Reply:
So the speaker component is entirely apart of the package.json that's precomiled in there already? If so, is there a reason it is there for the Linux installation steps?

Reply:
Well for starters - this is a node project, so telling people to install node is a bit unnecessary. If it turns out that `node-speaker` is needed, we can just add it back in to the `package.json`. I can't remember if it's currently in there or not, but there's no need for a separate install step. It was included in the instructions for linux, because on linux an additional package is required (`libasound2-dev`). That may be the case on Mac, in which case - by all means add it. Otherwise, just leave it out. :)